src/engine/N3Base/N3UIList.h

```diff
@@ -28,6 +28,7 @@ class CN3UIList : public CN3UIBase {
     D3DCOLOR            FontColor() { return m_crFont; }
     BOOL                FontIsBold() { return m_bFontBold; }
     BOOL                FontIsItalic() { return m_bFontItalic; }
+    CN3UIString *       GetChildStrFromList(std::string str);
```

Comment:
This should ideally be a const reference, since we have no reason to copy or change the string, i.e.:
```cpp
const std::string & str
```